conda/cli/install.py

```diff
@@ -145,8 +145,8 @@ def install(args, parser, command='install'):
 # $ conda update --prefix %s anaconda
 """ % prefix)
 
-    linked_dists = install_linked(prefix)
-    linked_names = tuple(ld.quad[0] for ld in linked_dists)
```

Comment:
OK, this concerns me. We're doing all this work to get rid of Dist objects, but it's not addressing the core issue, in my view: that we're depending on the _key_ to extract things like name, version, build.